app/client/components/billing/billing.tsx

```diff
@@ -211,11 +264,25 @@ const BillingRenderer = (apiResponse: MembersDataApiItem[]) => {
 export const Billing = (_: RouteComponentProps) => {
   return (
     <PageContainer selectedNavItem={NAV_LINKS.billing} pageTitle="Billing">
-      <MembersDatApiAsyncLoader
-        fetch={allProductsDetailFetcher}
+      <BillingDataAsyncLoader
+        fetch={billingFetcher}
         render={BillingRenderer}
         loadingMessage={`Loading your billing details...`}
       />
     </PageContainer>
   );
 };
+
+const billingFetcher = () =>
+  Promise.all([
+    allProductsDetailFetcher(),
+    fetch("/api/invoices", {
+      credentials: "include",
+      mode: "same-origin",
+      headers: {
+        [X_GU_ID_FORWARDED_SCOPE]: getScopeFromRequestPathOrEmptyString(
+          window.location.href
+        )
+      }
```

Comment:
is this needed or just copy/pasta from some other `fetch` call? This is for payment failure magic links, so not relevant here, I'd be inclined to remove it...

Reply:
Yep, copy pasta I'm afraid ... removed now.
![copypasta](https://user-images.githubusercontent.com/2510683/91037227-8a209100-e600-11ea-8a11-bac17e825f34.jpg)